app/src/main/java/com/cornellappdev/android/eatery/ui/screens/HomeScreen.kt

```diff
@@ -106,6 +105,15 @@ fun HomeScreen(
     val nearestEateries = homeViewModel.eateriesByDistance.collectAsState().value
     val eateriesApiResponse = homeViewModel.eateryFlow.collectAsState().value
     val filters = homeViewModel.filtersFlow.collectAsState().value
+    val homeScreenFilters = listOf(
+        Filter.FromEatery.North,
+        Filter.FromEatery.West,
+        Filter.FromEatery.Central,
+        Filter.FromEatery.Swipes,
+        Filter.FromEatery.BRB,
+        Filter.RequiresFavoriteEateries.Favorites,
+        Filter.FromEatery.Under10,
+    )
```

Comment:
Nitpick but can we move this into `homeViewModel`? It's ok if it's just some public constant there. I'd also want the same change to `SettingsScreen` where it's a bit more sus there since the `listOf` is declared like inline lol